### ISSUE TITLE
Access without browser plugin

### ISSUE THREAD

**Issue by ramzcode** [passbolt/passbolt_api#423]
Team,

Thanks for the wonderful product, is there a way to use directly from browser without a plugin?

It is difficult when moving between servers or browsers. Also login as another user literally make the old user not to access with going though the recovery process.

Is there any solution around this or planned in the future?

**Comment by stripthis** [comment]
Hello @ramzcode,

Currently there is no plan to make the web application work without browser extension because of few reasons. Please have a look at this blog post to learn more:
https://blog.passbolt.com/why-does-passbolt-require-an-extension-d1b189133b2

Best,

**Comment by ramzcode** [comment]
Great but switching to another user but even not authenticating as another user won't let the old user to connect back without recovery process. Is this also usual?

**Comment by stripthis** [comment]
One option would be to use browser profiles if you need to swith between different user / domain. This is a workaround, however there is a feature request to allow multiple user / domain on the same browser: https://community.passbolt.com/t/as-a-user-i-can-configure-passbolt-browser-extension-to-work-with-multiple-passbolt-instances/77/3

**Comment by ramzcode** [comment]
I read a lot about the same, BTW i don't think there are efforts for this as of now though many users have requested. This is really a pain for all who loves PassBoult.

If we can get this sorted then this is one of the top PM ever built in the open-source space.

**Comment by garrettboone** [comment]
Another option - and I just tested this again to make sure it works - is to use a PortableApps version of Firefox or Chrome. This installs the browser to a USB stick (I also installed the PortableApps app for app updating capability). Once the browser is at the passbolt site using the recovery link, it will prompt for the extension to be downloaded. At this point I found it keeps asking to download the extension (not sure why). I removed the `first=1` flag from the url and it took me straight to the recovery process. Once the extension is set up, the USB stick can be taken to another machine.

One drawback is that the stick is setup for a particular OS (not cross-platform). But if users are taking keys by USB anyway, maybe this is an option.
